Components/PlayerList/index.js

```diff
@@ -0,0 +1,89 @@
+import useSWR from "swr";
+import styled from "styled-components";
+import { useState } from "react";
+
+const StyledLetterList = styled.ul`
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+    margin-left: 0;
+    padding-right: 20px;
+    background-color: rgb(0, 0 , 0, 0.15)
+`
+
+const StyledButton = styled.button`
+    color: #0d48a0;
+    font-weight: 700;
+    border-radius: 20%;
+    background-color: #5bc0de;
+`
+
+const StyledPlayerList = styled.ul`
+    list-style: none; 
+    `;
+
+const StyledListItem = styled.li`
+align-items: center;
+font-size: 0.8rem;
+margin-bottom: 0.6rem;
+`;
+
+const Headline = styled.h1`
+  font-size: 2.5rem;
+  font-weight: bold;
+  margin-top: 0.5rem;  
+  font-family: "roboto", sans-serif;
```

Comment:
maybe define your fonts in a GlobalStyle - it will become quite messy if youre doing this in every component.

```suggestion
  font-family: "roboto", sans-serif;
```